runner/src/main.rs

```diff
@@ -221,39 +250,82 @@ fn format() -> Step {
     }
 }
 
+fn check_format() -> Step {
+    Step::Multiple {
+        name: "format".to_string(),
+        steps: vec![
+            run_check_license(),
+            run_check_todo(),
+            run_buildifier(FormatMode::Check),
+            run_prettier(FormatMode::Check),
+            run_markdownlint(FormatMode::Check),
+            run_embedmd(FormatMode::Check),
+            run_liche(),
+            run_cargo_fmt(FormatMode::Check),
+        ],
+    }
+}
+
 fn run_ci() -> Step {
```

Comment:
I think it is great that this command runs all the steps even if something fails in between. 
I suggest logging a summary of the failures or the list of failed steps in the end, so that one does not have to go through the entire log to locate the problem.

Reply:
Great idea! Created https://github.com/project-oak/oak/issues/1234 to track.

Reply:
that sweet issue number too